### PR TITLE
Features Table

### PR DESCRIPTION
**Pull request by Pro**
First draft for the features table. See https://github.com/open62541/open62541/issues/810

I took the supported services from `services.h`

For the lower table please check it carefully, since I just C&P it from node-opcua.

### CODE REVIEW

File: FEATURES.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+## Supported Features
+
+| __**Service**__             |                                 |                      | Comment              |
+|:----------------------------|:--------------------------------|:--------------------:|:---------------------|
+| Discovery Service Set       |                                 |                      |                      |
+|                             | FindServers()                   |  :white_check_mark:  |                      |
+|                             | FindServersOnNetwork()          | :first_quarter_moon: | Branch: feature_mdns |
+|                             | GetEndpoints()                  |  :white_check_mark:  |                      |
+|                             | RegisterServer()                |  :white_check_mark:  |                      |
+|                             | RegisterServer2()               |     :full_moon:      | Branch: master       |
```
Comment (markusgraube) [inline FEATURES.md]: What is RegisterServer2()?
This isn't a defined service in the spec and needs further explanation

Comment (jpfr) [inline FEATURES.md]: It comes with OPC UA v1.03

> This Service allows a Server to register its DiscoveryUrls and capabilities with a Discovery Server.
It extends the registration information from RegisterServer with information necessary for
FindServersOnNetwork. This Service will be called by a Server or a separate configuration utility.
Clients will not use this Service.

**Comment by jpfr** [comment]
Thanks for going ahead with this!
Merged.

File: FEATURES.md (hunk 1 above)
Comment (markusgraube) [inline FEATURES.md]: Okay, thank you for the info

**Pull request merged**